google/cloud/spanner_dbapi/utils.py

```diff
@@ -51,6 +53,15 @@ def __iter__(self):
 
 
 def backtick_unicode(sql):
+    """Check the SQL to be valid and split it by segments.
+
+    :type sql: str
+    :param sql: A SQL request.
+
+    :rtype: str
+    :returns: A SQL parsed by segments in unicode if initial
+             SQL is valid, initial string otherwise.
```

Comment:
Indentation should match the first description character of the previous line - in this case, `A`.